.travis.yml

```diff
@@ -5,19 +5,11 @@ services:
   - docker
 
 before_install:
-  # Add rethinkdb 2.3.6
-  - docker run --net="host" -p "29015:29015" -p "28015:28015" -d acalabs/rethinkdb:2.3.6
-
-env:
-  - TRAVIS_TEST=true
+  - docker run --net="host" -p "29015:29015" -p "28015:28015" -d rethinkdb:2.4
```

Comment:
I don't think we have to specify network host here. I forgot what the error log was but when running locally I have to leave that flag out

Reply:
I'm pretty sure because we are already exposing the ports, so we don't need the network host flag

Reply:
Not necessary here, although it doesn't cause problems

Reply:
I'm adding a github action so we can bid travis farewell